cypress/support/appObjectConfigs.js

```diff
@@ -59,7 +59,7 @@ class eventObject {
       this.eventSchemaResult = response.eventSchemaResult;
       this.eventTime = response.eventTime;
     } else {
-      assert(false, 'Event Response should be defined');
+      assert(false, 'Unable to find the event response');
     }
```

Comment:
So, is this a case when you get a response ? asking because it looks like it is not failing when there is no response received. It fails if response does not have the required parameters ? 
Shouldnt we
if(!response) {
 assert (platform did not trigger event)

}

Reply:
if there is no response it will fail in the linchpin step itself, if it is coming here that means we got some response but in that there is no event triggered response, so we can't diretly add if(!response) because FCA will send null value if there is no triggered response, so this if(!response) will never be true, so checking required values directly is the best thing.

Changed the log to platform did not trigger event